File: week 1/MiniMaxSum_2/MiniMaxSum.java
package MiniMaxSum_2;

import java.util.List;

public class MiniMaxSum {
    public void ResultMiniMaxSum(List<Integer> arr) {
        Integer min = arr.get(0);
        Integer max = arr.get(0);
        long sum = 0;

        for (Integer num : arr) {
            if (num > max) max = num;
            if (num < min) min = num;

            sum += num;
        }

        System.out.print((sum - max) + " ");
        System.out.print((sum - min) + " ");
    }

    public static void main(String[] args) {
        MiniMaxSum result = new MiniMaxSum();

        result.ResultMiniMaxSum(List.of(256741038, 623958417, 467905213, 714532089, 938071625));
    }
}

File: week 1/MiniMaxSum_2/MiniMaxSum.js
const MiniMaxSum = (arr = []) => {
    let min = arr[0]
    let max = arr [0]
    let sum = 0

    arr.forEach(num => {
        if (num > max) max = num 
        if (num < min) min = num

        sum += num
    })

    // Obtengo el minimo y el maximo de cada matriz
    // y simplemente hace falta restarlo dependiendo del caso
    // para el Max, tengo que restarle el mas pequeño y para el
    // Minimo tengo que restarlo el mas grande

    console.log(sum - max, sum - min)
}

MiniMaxSum([140638725, 436257910, 953274816, 734065819, 362748590])